server/src/test/java/org/elasticsearch/action/admin/cluster/storedscripts/GetStoredScriptResponseTests.java

```diff
@@ -0,0 +1,56 @@
+package org.elasticsearch.action.admin.cluster.storedscripts;/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import org.elasticsearch.common.xcontent.XContentParser;
+import org.elasticsearch.common.xcontent.XContentType;
+import org.elasticsearch.script.Script;
+import org.elasticsearch.script.StoredScriptSource;
+import org.elasticsearch.test.AbstractStreamableXContentTestCase;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.function.Predicate;
+
+public class GetStoredScriptResponseTests extends AbstractStreamableXContentTestCase<GetStoredScriptResponse> {
+
+    @Override
+    protected GetStoredScriptResponse doParseInstance(XContentParser parser) throws IOException {
+        return GetStoredScriptResponse.fromXContent(parser);
+    }
+
+    @Override
+    protected GetStoredScriptResponse createBlankInstance() {
+        return new GetStoredScriptResponse();
+    }
+
+    @Override
+    protected GetStoredScriptResponse createTestInstance() {
+        return new GetStoredScriptResponse(scriptSource());
+    }
+
+    @Override
+    protected Predicate<String> getRandomFieldsExcludeFilter() {
+        return s -> "script.options".equals(s);
+    }
+
+    private StoredScriptSource scriptSource() {
+        return new StoredScriptSource("lang", "code",
+            Collections.singletonMap(Script.CONTENT_TYPE_OPTION, XContentType.JSON.mediaType()));
```

Comment:
can we randomize how this instance gets created?

Reply:
agreed